src/css/base.css

```diff
@@ -1,19 +1,166 @@
 :root {
-  --primary-font: "Manrope", sans-serif;
+  --primary-font: 'Manrope', sans-serif;
   --black: #121417;
   --green: #4dc274;
   --gradient-green: linear-gradient(80deg, #4dc274 0%, #bee8cc 100%);
   --light-green: #d4fde1;
+  --lighter-green: #bee8cc;
   --gray: #f3f5f6;
   --white: #fff;
+  --hover-green: #38ae5f;
 }
 
-h1 {
+body {
   font-family: var(--primary-font);
+  color: var(--black);
+  background-color: var(--white);
+  font-size: 16px;
+  line-height: 1.5;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: var(--black);
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.section {
+  padding-bottom: 80px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-width: 1024px;
+  min-width: 320px;
+  margin: 0 auto;
+}
+
+.section-title {
+  position: relative;
+  font-size: 32px;
   font-weight: 700;
-  font-size: 20px;
-  color: #121417;
+  line-height: 1.2;
+  margin-bottom: 16px;
+}
+
+.section-title::after {
+  content: '';
+  position: absolute;
+  display: block;
+  width: 144px;
+  height: 20px;
+  background-color: var(--lighter-green);
+  border-radius: 8px;
+  right: 0;
+  bottom: 0;
+  z-index: -1;
 }
 
-/* .container {
-} */
\ No newline at end of file
+.section-subtitle {
+  font-size: 14px;
+  opacity: 0.5;
+  margin-bottom: 40px;
+  max-width: 665px;
+  text-align: center;
+}
+
+.button {
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 1.7;
+  color: var(--white);
+  background-color: var(--green);
+  border-radius: 12px;
+  padding: 16px 32px;
```

Comment:
for mobile vertical paddings should be 14px and for tablet/desktop 16px

Reply:
Done